crates/swift-integration-tests/src/shared_types/shared_enum.rs

```diff
@@ -9,8 +9,15 @@ mod ffi {
         fn reflect_enum_with_no_data(arg: EnumWithNoData) -> EnumWithNoData;
     }
 
+    extern "Rust" {
+        #[swift_bridge(Equatable)]
+        type Foo;
```

Comment:
Let's use a more descriptive name since if we use names like `Foo` the global namespace on the Swift side will get polluted with types that have names that aren't useful. 

Maybe `type OpaqueRustForEnumTest;`